### PR TITLE
Fix stats issues after Cere node update

### PR DESCRIPTION
**Pull request by DmitriyMolch**

### CODE REVIEW

File: frontend/pages/validator/_id/index.vue [hunk 1]
```diff
@@ -214,22 +214,6 @@
               />
             </div>
           </div>
-          <div class="row">
```
Comment (AndreiNavoichyk) [inline frontend/pages/validator/_id/index.vue:217]: Why did you remove this?

**Review by AndreiNavoichyk — APPROVED** [review]

**Review by AndreiNavoichyk — COMMENTED** [review]
Check comments.

File: frontend/pages/validator/_id/index.vue (hunk 1 above)
Comment (DmitriyMolch) [reply frontend/pages/validator/_id/index.vue:217]: Removed this because this row is empty on the page.  `Gevernence` component was not imported in the initial version of UI so it wasn't rendered but caused a warming in the development mode.

**Review by AndreiNavoichyk — APPROVED** [review]
